pyhelper_utils/utils.py

```diff
@@ -0,0 +1,67 @@
+import subprocess
```

Comment:
This all repo is one big utils, we need to find another name for the file (not utils.py).
commands, executes etc...

Reply:
`shell` is a good name for this file.

Reply:
done.